### PR TITLE
Add Midnight color palette to SpaceKit

### PR DESCRIPTION
**Pull request by jglovier**
This PR adds the midnight color palette to SpaceKit for use as a dark mode theme color base. 🌃 

| Preview |
|---|
|image|

### CODE REVIEW

File: stories/Color.stories.js [hunk 1]
```diff
@@ -55,6 +55,6 @@ storiesOf("Color", module)
     <ColorStory
       title="Alternate Colors"
       description="The alternate colors are rounding out the set of colors we have in the system, but will likely not be used that often. A common use case for these colors will be data visualizations."
-      array={["orange", "yellow", "purple", "blilet"]}
+      array={["orange", "yellow", "purple", "blilet", "midnight"]}
```
Comment (jglovier) [inline stories/Color.stories.js:58]: @justinanastos my only question is where this color should really live? I just stuck it here for now, but considering it's specifically for a UI theme...does it belong elsewhere?

**Comment by jglovier** [comment]
What else do I need to change with this PR? Should I be bumping a version number? Anything else?

File: stories/Color.stories.js (hunk 1 above)
Comment (justinanastos) [reply stories/Color.stories.js:58]: Nope! I have no preference here. I'd like to revamp this eventually but I have no idea how to do that. This looks good to me.

**Comment by justinanastos** [comment]
> What else do I need to change with this PR? Should I be bumping a version number? Anything else?

@jglovier Versions get bumped automatically depending on the label you choose 😃

**Comment by apollo-bot2** [comment]
:rocket: PR was released in [`v5.4.0`](https://github.com/apollographql/space-kit/releases/tag/v5.4.0) :rocket:

**Pull request merged**